tests/cases/compiler/classMergedWithInterfaceMultipleBasesNoError.ts

```diff
@@ -0,0 +1,10 @@
+interface Bar { }
+interface Baz { }
+interface Q { }
+interface Foo extends Bar, Baz { }
+class Foo { }
+
+export default class extends Foo {
+    readonly observer = this.handleIntersection;

```

Comment:
This should still raise a use before define error (maybe it's a `--strict` thing?), but I guess it should still compile at any rate.

Reply:
There's an [error baseline](https://github.com/weswigham/TypeScript/blob/11373d4e8e9b9fce6f8800d1b0788748d8d909f2/tests/baselines/reference/classMergedWithInterfaceMultipleBasesNoError.errors.txt)

Reply:
Oh, I missed it (the `NoError` threw me off) 🙇‍♀️

Reply:
Kinda meant no error as in doesn't crash. Maybe not the clearest term.